ts/theme/components/Button.ts

```diff
@@ -39,7 +39,7 @@ export default (): Theme => ({
     },
     "UIComponent.IconFont": {
       fontSize: variables.btnXSmallIconSize,
-      paddingRight: 4
+      paddingRight: 8
```

Comment:
In this way you are changing the padding for `all buttons` in the whole.
Is this change safe?
See what happens if I set this value to 30 (to make it evident)

<img src="https://user-images.githubusercontent.com/822471/105333482-10ec9080-5bd6-11eb-85bb-209c401e4db5.jpg" height="600"/>

Perhaps it should be changed locally only in the button you refactored

```typescript
const HelpButton: FC<HelpButtonProps> = ({ onShowHelp }) => (
  <ButtonDefaultOpacity
    hasFullHitSlop
    style={{padding: 8}}
    onPress={onShowHelp}
    transparent={true}
    accessibilityLabel={I18n.t(
      "global.accessibility.contextualHelp.open.label"
    )}
    accessibilityHint={I18n.t("global.accessibility.contextualHelp.open.hint")}
  >
    <IconFont name={"io-question"} />
  </ButtonDefaultOpacity>
);
```